src/main/fc/fc_tasks.c

```diff
@@ -586,6 +592,15 @@ cfTask_t cfTasks[TASK_COUNT] = {
     },
 #endif
 
+#ifdef RUNCAM_SPLIT_SUPPORT
+    [TASK_RCSPLIT_SUPPORT] = {
+        .taskName = "RCSPLIT",
+        .taskFunc = rcsplitProcess,
+        .desiredPeriod = TASK_PERIOD_HZ(100),        // 100 Hz, 10ms
```

Comment:
Not sure about RC mode activation, but without it much lower frequency and low priority should do (10Hz?)